pygments/lexers/arturo.py

```diff
@@ -0,0 +1,312 @@
+            include('builtin-predicate-functions'),
+            (words((
+                'abs', 'acos', 'acosh', 'acsec', 'acsech', 'actan', 'actanh',
+                'add', 'after', 'alphabet', 'and', 'angle', 'append', 'arg',
+                'args', 'arity', 'array', 'as', 'asec', 'asech', 'asin',
+                'asinh', 'atan', 'atan2', 'atanh', 'attr', 'attrs', 'average',
+                'before', 'benchmark', 'blend', 'break', 'builtins1',
+                'builtins2', 'call', 'capitalize', 'case', 'ceil', 'chop',
+                'chunk', 'clear', 'close', 'cluster', 'color', 'combine',
+                'conj', 'continue', 'copy', 'cos', 'cosh', 'couple', 'csec',
+                'csech', 'ctan', 'ctanh', 'cursor', 'darken', 'dec', 'decode',
+                'decouple', 'define', 'delete', 'desaturate', 'deviation',
+                'dictionary', 'difference', 'digest', 'digits', 'div', 'do',
+                'download', 'drop', 'dup', 'e', 'else', 'empty', 'encode',
+                'ensure', 'env', 'epsilon', 'escape', 'execute', 'exit', 'exp',
+                'extend', 'extract', 'factors', 'false', 'fdiv', 'filter',
+                'first', 'flatten', 'floor', 'fold', 'from', 'function',
+                'gamma', 'gcd', 'get', 'goto', 'hash', 'help', 'hypot', 'if',
+                'in', 'inc', 'indent', 'index', 'infinity', 'info', 'input',
+                'insert', 'inspect', 'intersection', 'invert', 'join', 'keys',
+                'kurtosis', 'last', 'let', 'levenshtein', 'lighten', 'list',
+                'ln', 'log', 'loop', 'lower', 'mail', 'map', 'match', 'max',
+                'maybe', 'median', 'min', 'mod', 'module', 'mul', 'nand',
+                'neg', 'new', 'nor', 'normalize', 'not', 'now', 'null', 'open',
+                'or', 'outdent', 'pad', 'panic', 'path', 'pause',
+                'permissions', 'permutate', 'pi', 'pop', 'pow', 'powerset',
+                'powmod', 'prefix', 'print', 'prints', 'process', 'product',
+                'query', 'random', 'range', 'read', 'relative', 'remove',
+                'rename', 'render', 'repeat', 'replace', 'request', 'return',
+                'reverse', 'round', 'sample', 'saturate', 'script', 'sec',
+                'sech', 'select', 'serve', 'set', 'shl', 'shr', 'shuffle',
+                'sin', 'sinh', 'size', 'skewness', 'slice', 'sort', 'split',
+                'sqrt', 'squeeze', 'stack', 'strip', 'sub', 'suffix', 'sum',
+                'switch', 'symbols', 'symlink', 'sys', 'take', 'tan', 'tanh',
+                'terminal', 'to', 'true', 'truncate', 'try', 'type', 'union',
+                'unique', 'unless', 'until', 'unzip', 'upper', 'values', 'var',
+                'variance', 'volume', 'webview', 'while', 'with', 'wordwrap',
+                'write', 'xnor', 'xor', 'zip'
+            ), prefix=r'\b', suffix=r'\b'), Name.Builtin)
+        ],
+            'builtin-predicate-functions': [
+                (
+                    r'\b('
```

Comment:
use words() again...

Reply:
In this case, `words` aren't working to me.
It's the reason I used this handmade regex.

Reply:
- [x] [replace with words function](https://github.com/pygments/pygments/pull/2259/commits/485b840311819421b1744f630f9490baaa53dda6)

Reply:
- [x] [replace hand-made regex with words function](https://github.com/pygments/pygments/pull/2259/commits/2a5b4d602f07a9bd71910e73134e707b8bcb5a64)

I done! But I got a question. 
When I use it, this doesn't match the `?` at the end on syntax highlight. Look:
![image](https://user-images.githubusercontent.com/78623871/197778278-a043adb0-8666-4818-9a44-2524b8055b0f.png)

As You can see, `or` is matched as `Name.Builtin`, but, `?` mark - that is part of `or` - doesn't. 
So, `or?` should be matched as a `Name.Builtin`, not just `or`.

Just as labels are:
![image](https://user-images.githubusercontent.com/78623871/197778991-15b610bb-a759-4c2f-ad59-12fd2c8e6427.png)

Reply:
- [x] [add missing r string prefix](https://github.com/pygments/pygments/pull/2259/commits/aa87683cb1f4ec9bd3ed728d4d63b5f32a1e3b3a)
- [x] [add missing commas](https://github.com/pygments/pygments/pull/2259/commits/2ab3cf7c2f62506080f90c3e62013a5f8b7deca4)